index.html

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+	<meta charset="UTF-8">
+	<meta http-equiv="X-UA-Compatible" content="IE=edge">
+	<meta name="viewport" content="width=device-width, initial-scale=1.0">
+	<link rel="stylesheet" href="style.css">
+	<title>Portfolio</title>
+</head>
+<body>
+	<section class="screen_container">
+		<header>
+			<nav>
+				<section class="toolbar_container">
+					<p class="Title_logo">My Logo</p>
```

Comment:
- [ ] Please use `a` tag for your logo. This is for three reasons: convention, semantics, and user experience(UX). [Check this link](https://ux.stackexchange.com/questions/81727/why-is-it-standard-for-a-website-logo-to-navigate-to-the-home-page) for more info.

- [ ] Kindly move the logo outside of the `nav` tag. `nav` tag it uses to wrap navigation links on the webpage.